projects/plugins/jetpack/extensions/plugins/ai-content-lens/editor.js

```diff
@@ -4,7 +4,7 @@ import { store as editorStore } from '@wordpress/editor';
 import { addFilter } from '@wordpress/hooks';
 import debugFactory from 'debug';
 import metadata from '../../blocks/ai-assistant/block.json';
-import { isPossibleToExtendBlock } from '../../blocks/ai-assistant/extensions/ai-assistant';
+import { isPossibleToExtendBlock } from '../../blocks/ai-assistant/extensions/lib/is-possible-to-extend-block';
```

Comment:
This change seems to be weird. It used the `isPossibleToExtendBlock` from `ai-assistant/extensions/ai-assistant` instead of `ai-assistant/extensions/lib/is-possible-to-extend-block` that checks the following requirements
* isBlockRegistered
* isAiAssistantSupportEnabled
* isUserConnected
* whether the current plan supports ai assistant feature or not
* whether the specific block is hidden or not

And it didn't require `blockName` before.

However, with this change, it leads to the function always returning false because the blockName is missing...
```
if ( ! EXTENDED_BLOCKS.includes( blockName ) ) {
	return false;
}
```

Reply:
Excellent catch there, Arthur!
We need to change the function that is called to check if we should enable the excerpt or not, as `isPossibleToExtendBlock` should be only for block extensions.
Taking a look at your PR now.